src/manager/source_drive_common.hpp

```diff
@@ -62,9 +62,9 @@ class DriveYamlParam
         YamlRead<std::string>(config["ros"], "ros_send_correction_topic",  driver_param.input_param.ros_send_correction_topic, NULL_TOPIC);
         YamlRead<std::string>(config["ros"], "ros_send_firetime_topic",    driver_param.input_param.ros_send_firetime_topic, NULL_TOPIC);
         YamlRead<std::string>(config["ros"], "ros_recv_correction_topic",  driver_param.input_param.ros_recv_correction_topic, NULL_TOPIC);  
-        YamlRead<std::string>(config["ros"], "ros_send_imu_topic",         driver_param.input_param.ros_send_imu_topic, NULL_TOPIC);              
-        return true;
+        YamlRead<std::string>(config["ros"], "ros_send_imu_topic",         driver_param.input_param.ros_send_imu_topic, NULL_TOPIC);
+	YamlRead<float>(       config["ros"], "ros_imu_publish_rate",       driver_param.input_param.ros_imu_publish_rate, -1.0);	
+	return true;
     }
-
     
-};
\ No newline at end of file
+};
```

Comment:
Add blank line for consistency
```suggestion
};

```

Reply:
No new line was present before modifications